translate_zh_simple.py

```
@@ -0,0 +1,1573 @@
+    # Browser Pane
+    "Could not open the downloaded file. It may have been moved or deleted.": "无法打开下载的文件。文件可能已被移动或删除。",
+    "Could not show the downloaded file. It may have been moved or deleted.": "无法显示下载的文件。文件可能已被移动或删除。",
+    "Downloading paused": "下载已暂停",
+    "Download failed": "下载失败",
+    "The downloaded file path is unavailable.": "下载的文件路径不可用。",
+    
+    # Crash Report
+    "Attach recent diagnostic logs": "附加最近的诊断日志",
+    "Failed to send crash report. Diagnostic ticket {{value0}} was uploaded but not linked.": "发送崩溃报告失败。诊断工单 {{value0}} 已上传但未关联。",
+    "Sends a capped redacted log bundle with the report.": "随报告发送有限制的脱敏日志包。",
+    "No automatic crash report was captured. You can still send details and include recent diagnostic logs when available.": "未捕获到自动崩溃报告。您仍然可以发送详细信息，并在可用时包含最近的诊断日志。",
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)